src/table/utils/get-visible-height.ts

```diff
@@ -0,0 +1,19 @@
+import { initialPageInfo } from '../../nebula-hooks/use-pagination-table';
+import { TableLayout, ViewService } from '../../types';
+
+const getVisibleHeight = (
+    visibleRowEndIndex: number,
+    visibleRowStartIndex: number,
+    layout: TableLayout,
+    viewService: ViewService
+  ) => {
+    if (visibleRowEndIndex < 0) return 0;
+  
+    const totalRowCount = layout.qHyperCube.qSize.qcy;
+    const visualRowsPerPage = viewService.rowsPerPage || initialPageInfo.rowsPerPage;
```

Comment:
I think you mean `visible`RowsPerPage and not `visual`RowsPerPage right?

Reply:
This comes from https://github.com/qlik-oss/sn-table/pull/1035. I just moved the logic to another file to make it easier to write unit tests and mocking.